core/esp_spi.c

```diff
@@ -269,7 +283,7 @@ size_t spi_transfer(uint8_t bus, const void *out_data, void *in_data, size_t len
         size_t offset = i * _SPI_BUF_SIZE;
         _spi_buf_transfer(bus, (const uint8_t *)out_data + offset,
             in_data ? (uint8_t *)in_data + offset : NULL, buf_size, e, word_size);
-        if (blocks) _rearm_extras_bit(bus, false);
+       _rearm_extras_bit(bus, false);
```

Comment:
Could you consider adding a comment to `_rearm_extras_bit` on what it does. It seems to be to toggle off the address in the middle of transfer needing multiple blocks, so the address is only sent at the start?

Is using the full `_SPI_BUF_SIZE` ok? I recall mention of a possible hardware problem using a size of 64 for reads, and it notes that the esp rom code uses a size of 32 bytes, see http://www.esp8266.com/viewtopic.php?f=13&t=10843 Might it be prudent to change the size to 32, or at least 60?